app/(server)/api/events/[slug]/route.ts

```diff
@@ -75,9 +87,19 @@ export async function DELETE(
     );
   }
 
-  const response = await getCurrentAuthenticated(requestToken.value);
+  const response: AuthType.CurrentAuthResponse = await InternalApiFetcher.get(
+    '/api/auth/current',
+    {
+      headers: {
+        Authorization: `Bearer ${requestToken?.value || ''}`,
+      },
+      cache: 'no-cache',
+    }
+  );
```

Comment:
should be replaced with a server side function

Reply:
Updated on this commit [36a5218](https://github.com/inlivedev/inlive-room/pull/128/commits/36a52182807119a594c0a94cf22615d04a22cba9)